packit/cli/update.py

```diff
@@ -31,4 +31,8 @@ def update(config, dist_git_path, dist_git_branch, repo):
     """
     package_config = get_local_package_config(directory=repo.working_dir)
     api = PackitAPI(config=config, package_config=package_config)
-    api.sync_release(dist_git_branch, dist_git_path=dist_git_path)
+    try:
+        api.sync_release(dist_git_branch, dist_git_path=dist_git_path)
+    except PackitFedpkgException as pfe:
```

Comment:
Could we catch the generic packitexception here?

Reply:
I have nothing against it.